README.md

```diff
@@ -33,59 +32,62 @@ To check if your system supports virtualization, you can use the following comma
 if egrep -c '(vmx|svm)' /proc/cpuinfo > /dev/null; then echo "Virtualization support found"; fi
 ```
 
-On Linux, Uhyve depends on the virtualization solution [KVM](https://www.linux-kvm.org/page/Main_Page) (Kernel-based Virtual Machine).
+Uhyve on Linux depends on the virtualization solution [KVM (Kernel-based Virtual Machine)](https://www.linux-kvm.org/page/Main_Page).
 If the following command gives you some output, you are ready to go!
 
 ```sh
 lsmod | grep kvm
 ```
 
-**NOTE:** If in case the above steps don't work, make sure to check in your BIOS settings that virtualization is enabled there.
+> [!NOTE]
+> If the above steps don't work, make sure that you have enabled virtualization in your UEFI/BIOS settings.
 
 ### macOS
 
-**Disclaimer:** Currently, Uhyve is mainly developed for Linux.
-The macOS version has not been tested extensively and does not support all features of the Linux version.
+> [!WARNING]
+> Currently, Uhyve is mainly developed for Linux.
+> The macOS version has not been tested extensively and does not support all features of the Linux version.
 
-Apple's *Command Line Tools* must be installed.
-The following terminal command installs these tools *without* Apple's IDE Xcode:
+You can install Apple's [Xcode Command Line Tools](https://developer.apple.com/xcode/resources) using the following command:
 
 ```sh
 xcode-select --install
 ```
 
 Additionally, the included hypervisor bases on the [Hypervisor Framework](https://developer.apple.com/documentation/hypervisor) depending on OS X Yosemite (10.10) or newer.
-To verify if your processor is able to support this framework, run the following in your Terminal:
 
-```sh
-sysctl kern.hv_support
-```
+> [!NOTE]
+> To verify if your processor is able to support this framework, run the following in your Terminal:
+>
+> ```sh
+> sysctl kern.hv_support
+> ```
```

Comment:
Oh, didn't see this earlier. I don't think this needs extra highlighting with a note. Especially, since the following line is also not hightlighted